source/_patterns/utilities/typography/headings.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="container-fluid">
```

Comment:
Please remove everything from here, that's not the headings and also remove the `mb-pf-md` class. All the extra information should go just to the md file.

Leave just this:

```html
<h1 class="h1">Heading level 1</h1>
<h2 class="h2">Heading level 2</h2>
<h3 class="h3">Heading level 3</h3>
<h4 class="h4">Heading level 4</h4>
```